src/containers/organizations/reducers/OrganizationsReducer.js

```diff
@@ -37,6 +37,7 @@ const INITIAL_STATE = Immutable.fromJS({
   isFetchingOrgs: false,
   isSearchingOrgs: false,
   isSearchingUsers: false,
+  isConfirmingDeletion: false,
```

Comment:
I don't think the modal will be hidden after the deletion takes place. You only ever hide it on `HIDE_DELETE_MODAL`, which only happens when the cancel button is clicked. You add `state.set('isConfirmingDeletion', false);` inside `DELETE_ORG_SUCCESS`.